compressor/tests/test_storages.py

```diff
@@ -15,16 +15,16 @@
 
 class GzipStorage(LazyObject):
     def _setup(self):
-        self._wrapped = get_storage_class(
-            "compressor.storage.GzipCompressorFileStorage"
-        )()
+        self._wrapped = storages.create_storage({
```

Comment:
That's not a documented API is it ? Same below.

Reply:
No, not yet. Same as setting the backends on the StorageHandler, I want to make this documented. (We need it; so will others.)